python/gui/auto_generated/providers/qgsdbsourceselectbase.sip.in

```diff
@@ -28,16 +28,22 @@ Constructor
 %End
 
   protected:
-    void setSourceModel( QgsAbstractDbTableModel *model );
+    void init( QgsAbstractDbTableModel *model, QItemDelegate *delegate = 0 );
```

Comment:
Maybe just stub out setSourceModel to avoid the API break?

Reply:
there is no API break, the class is new

Reply:
Ah, of course!